requirements-dev.txt

```diff
@@ -1,5 +1,4 @@
 black>=22.1.0
-responses < 0.19.0
```

Comment:
Does the issue still exist in version `0.19.0` or was it just not compatible with something else? If there is an issue in `0.19.0` I would suggest setting `responses>=0.20.0`.

Reply:
It works with both `0.19.0` and `0.20.0` when I test it locally.

Reply:
Ok, then it was probably some incompatibility with another package. Then we can leave it without restrictions.